### DISCUSSION TITLE
Devices and entities not available

### DISCUSSION THREAD

**Discussion by mikele983** [home-assistant/frontend#22625] [Automations / scripts / scenes]
I have an ecovacs deboot 715 robot. Once the installation is complete (ecovacs integration module), the system shows me both the entities and the "Not available" devices. I logged in via cloud mode. I currently use Home Assistant with a virtual machine on a PC. What could be the problem? 
![Schermata 2024-11-01 alle 16 37 41](https://github.com/user-attachments/assets/4827a942-824a-46cf-912d-a5590786a631)

**Comment by karwosts** [comment]
This is not a support forum, I think you'll have better luck here (and also this isn't even related to frontend). 

https://community.home-assistant.io/
